apps/dashboard/components.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "https://ui.shadcn.com/schema.json",
+  "style": "new-york",
+  "rsc": false,
+  "tsx": true,
+  "tailwind": {
+    "config": "tailwind.config.js",
+    "css": "src/index.css",
+    "baseColor": "neutral",
+    "cssVariables": true,
+    "prefix": ""
+  },
+  "aliases": {
+    "components": "@/components",
+    "utils": "@/utils/ui",
+    "ui": "@/components/primitives",
+    "lib": "@/utils",
+    "hooks": "@/hooks"
```

Comment:
`"ui": "@/components/primitives",` and `"lib": "@/utils",` are confusing to me.

Should we have instead: `"components/primitives": "@/components/primitives",` and `"lib": "@/lib", wdyt?

Reply:
Left side needs to stay the same for the cli to work correctly.

Reply:
ahh ok that's the structure that is forced by shadcn